restlicsharpclient/restliclienttest/RestClientTests.cs

```diff
@@ -93,9 +93,9 @@ public void SerializeAndDeserializeRecordWithEnum()
             greetingBuilder.message = "Hello, Serialize test!";
             Greeting g = greetingBuilder.Build();
 
-            string serialized = JsonConvert.SerializeObject(g);
+            string serialized = System.Text.Encoding.UTF8.GetString(DataUtil.SerializeObject(g));
```

Comment:
Provide two methods: SerializeObject (serializing object to string, paired with DeserializeObject), and the other is SerializeObjectToBytes (used in client).